lms/static/js/ccx/schedule.js

```diff
@@ -251,9 +265,25 @@ var edx = edx || {};
 	    }
 	},
 
+	valid_dates: function(start, due) {
+          var errorMessage;
+          if (_.isEmpty(start) && !_.isEmpty(due)) {
+            errorMessage = "Please enter valid start date and time";
```

Comment:
Also, you have error checking for an empty `start` date, but you don't have any error checking for am empty `due` date. Is that intentional? If so, please add a comment indicating that. Right now, it feels like that case was simply overlooked.